backend/src/main/java/com/redhat/cloud/notifications/routers/internal/errata/ErrataUserPreferencesMigrationResource.java

```diff
@@ -0,0 +1,96 @@
+package com.redhat.cloud.notifications.routers.internal.errata;
+
+import com.redhat.cloud.notifications.Constants;
+import com.redhat.cloud.notifications.auth.ConsoleIdentityProvider;
+import io.quarkus.logging.Log;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.inject.Inject;
+import jakarta.validation.constraints.NotNull;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.MediaType;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+import org.jboss.resteasy.reactive.RestForm;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+@Path(Constants.API_INTERNAL + "/team-nado")
+@RolesAllowed(ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN)
+public class ErrataUserPreferencesMigrationResource {
+    private static final String USERNAME_KEY = "username";
+    private static final String ORG_ID_KEY = "org_id";
+    private static final String[] CSV_HEADERS = {USERNAME_KEY, ORG_ID_KEY};
+
+    @Inject
+    ErrataMigrationRepository errataMigrationRepository;
+
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("/migrate/csv")
```

Comment:
As discussed in Slack with Teamnado, it seems we won't need this endpoint.

Reply:
Removed!